### PR TITLE
Update 1.15 release notes for tensorrt and mixed precision

### PR DESCRIPTION
**Pull request by nluehr**

### CODE REVIEW

**Review by aaroey — APPROVED** [review]
Thanks for the update! TRT related changes LGTM.

**Comment by aaroey** [comment]
@reedwm @goldiegadde

File: RELEASE.md [hunk 1]
```diff
@@ -12,6 +12,18 @@ This enables writing forward compatible code: by explicitly importing either `te
 * Adds `enable_tensor_equality()`, which switches the behavior such that: 
   * Tensors are no longer hashable.
   * Tensors can be compared with == and !=, yielding a Boolean Tensor with element-wise comparison results. This will be the default behavior in 2.0.
+* Auto Mixed-Precision graph optimizer simplifies converting models to float16 for acceleration on Volta and Turing Tensor Cores. This feature can be enabled by wrapping an optimizer class with `tf.train.experimental.enable_mixed_precision_graph_rewrite()`.
```
Comment (reedwm) [inline RELEASE.md:15]: Wasn't this in 1.14?

**Review by reedwm — CHANGES_REQUESTED** [review]

File: RELEASE.md (hunk 1 above)
Comment (nluehr) [reply RELEASE.md:15]: yes, but it was never relnoted. I can begin the note with "Starting from TensorFlow 1.14 ..." if you like.

Comment (reedwm) [reply RELEASE.md:15]: I think this is fine. No need for the "Starting from TensorFlow 1.14"

**Review by reedwm — APPROVED** [review]

**Comment by mihaimaruseac** [comment]
This is an automated account for release notes management. We cannot merge PRs here.

**Pull request closed without merging**